osmosis-1/osmosis.zone_assets.json

```diff
@@ -3439,17 +3439,24 @@
       "_comment": "BackBone Labs Liquid Staked OSMO $bOSMO"
     },
     {
-      "chain_name": "pundix",
-      "base_denom": "ibc/55367B7B6572631B78A93C66EF9FDFCE87CDE372CC4ED7848DA78C1EB1DCDD78",
-      "path": "transfer/channel-12618/transfer/channel-0/eth0x0FD10b9899882a6f2fcb5c371E17e70FdEe00C38",
+      "chain_name": "fxcore",
+      "base_denom": "ibc/46D8D1A6E2A80ECCB7CA6663086A2E749C508B68DA56A077CD26E6F4F9691EEE",
```

Comment:
This is incorrect. It's be base denom on Osmosis, but that's not what's needed here. The base denom on fxcore is 'eth0x0FD10b9899882a6f2fcb5c371E17e70FdEe00C38'